### DISCUSSION TITLE
关于 Crumbs 面包屑组件开发相关

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#39] [General]
## 介绍

面包屑用于显示当前页面的路径，快速返回之前的任意页面。

## 组件

`Crumbs` 面包屑组件内部其实是需要两个组件联合使用

- `f-crumbs` 容器组件
- `f-crumbs-item` 每一项元素

## Attributes

**f-crumbs**

- 自定义分隔符
- 默认文字颜色
- 当前页文字颜色（当前页文字加粗）
- ……

**f-crumbs-item**

- 跳转的路径
- 是否加粗
- ……

## 难点

面包屑组件是需要路由跳转的，而用户是不一定安装了 `vue-router`，如果使用 `$router` 跳转的话，就会报错，那么你需要考虑如何解决跳转的问题了。

## 设计稿

https://www.figma.com/file/xkjPUkMNbhFAVOu90OhKih/Crumbs-%E9%9D%A2%E5%8C%85%E5%B1%91

**Comment by TerryMinH** [comment]
点击面包屑暴露外面对应事件不就行了么 。具体如何处理用用户自己决定

**Reply by ghost** [reply]
我们需要内置一个路由跳转的功能， 这样可以提升用户在使用组件过程中的体验

**Comment by Tyh2001** [comment]
其实在 vue 中有内置的 api 可以解决的
